src/lib.rs

```diff
@@ -137,6 +146,17 @@ impl Expr {
         }
     }
 
+    /// Converts this expression to a SqlQuery Struct
```

Comment:
Intra-doc links are pretty cheap and easy so I prefer to use them when possible:

```suggestion
    /// Converts this expression to a [SqlQuery] struct
```